sklearn/ensemble/_base.py

```diff
@@ -54,10 +54,10 @@ def _set_random_states(estimator, random_state=None):
         parameters.
 
     random_state : int or RandomState, default=None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        Pseudo-random number generator to control the generation of the random
+        integers.
```

Comment:
Nit:

We can continue with the next sentence without a newline:


```
integers. Pass an int for...
```

Reply:
Thanks @thomasjpfan , I addressed your comment.